src/bors/event.rs

```diff
@@ -1,5 +1,6 @@
 use crate::database::{WorkflowStatus, WorkflowType};
 use crate::github::{CommitSha, GithubRepoName, GithubUser, PullRequest, PullRequestNumber};
+use chrono::Duration;
```

Comment:
Let's use `std::time::Duration` here, to avoid the casting between chrono and std.

Reply:
i used chrono beacuase it is what used by github.